iidm/iidm-extensions/src/test/java/com/powsybl/iidm/network/extensions/MeasurementTest.java

```diff
@@ -0,0 +1,119 @@
+package com.powsybl.iidm.network.extensions;
+
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.junit.jupiter.api.Test;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.ThreeSides;
+
+class MeasurementTest {
```

Comment:
You should add a Javadoc comment with the `@author` clause (see other files for example).

Reply:
Done